test/openAPI/features/support/helpers/helpers.js

```diff
@@ -252,4 +252,18 @@ module.exports = {
       BirthCertificateID: { type: 'string' },
     },
   },
+  dataListReadEndpoint: 'data/{registryname}/{versionnumber}',
+  dataListResponseSchema: {
+    type: 'object',
+    properties: {
+      count: { type: 'number' },
+      next: { type: 'string' },
+      previous: { type: ['string', 'null'] },
+      results: {
+        type: 'array',
+          items: this.dataExampleSchema
+      }
+    },
+    required: ['count', 'results']
+  }
```

Comment:
Can I have one suggestion? Can you please move this lines above? Like you see this file is divide by some sections and the last one is "shares" for every schemas that are used many times. So have this code above after `// data_list` comment will help keep this file tidy ;)

Reply:
updated